### ISSUE TITLE
Support MSIF 

### ISSUE THREAD

**Issue by asm0dey** [Komet/MediaElch#1747]
**Is your feature request related to a problem? Please describe.**
For quite some time already, Kodi forces us to use the movie set information folder: https://kodi.wiki/view/Movie_set_information_folder, it would be nice to support it

**Describe the solution you'd like**
I would propose to add a third choice to the dropdown box of movie set image storage, called "Movie Set Information Folder", that will follow the Kodi naming conventions by default

**Describe alternatives you've considered**
Currently I have to download the artwork manually and put it into the described folder

**Comment by bugwelle** [comment]
Thanks for opening this feature request. Indeed, the "movie set" feature needs an overhaul. It doesn't work properly and there is a lot to add to.

Please note my comments in https://github.com/Komet/MediaElch/issues/1710, though. If there is anyone else who wants to implement this feature, I'll gladly support them and review code, etc.

Regards,
Andre

**Comment by asm0dey** [comment]
Yeah, I'm not much help in C++, my languages are Java and other JVM languages, and also Python and JS to some degree.
I've tried to look into the code when encountered an issue, but quickly understood that I'm not good enough. Sorry :(
